src/bluetoothdevice.h

```diff
@@ -53,42 +53,38 @@ class bluetoothdevice : public QObject {
 
     /**
      * @brief currentSpeed Gets a metric object for getting and setting the speed. Units: km/h
-     * @return
      */
     virtual metric currentSpeed();
 
     /**
-     * @brief currentPace Gets the current pace. Units: ?
-     * @return
+     * @brief currentPace Gets the current pace. Units: time per km
      */
     virtual QTime currentPace();
 
     /**
-     * @brief currentInclination
-     * @return
+     * @brief currentInclination The current inclination. Expected range: 0 degrees (horizontal) to 90 degrees (vertical)
```

Comment:
actually some treadmill can have also -15%. The max I guess it will never go more than 25% but of course any value could be accepted. Every device module would limit this to its max.

Reply:
I've changed the wording to use inclination as vertical:horizontal instead of degrees, as well as being vague about limits.